### ISSUE TITLE
epoch settings for 4 gpus or 2 gpus

### ISSUE THREAD

**Issue by liuliu66** [open-mmlab/mmdetection#1110]
Hi,

if I want to use 4 gpus or 2 gpus to reproduce the detection result, do I need only change the initial learning rate from 0.02 to 0.01 or 0.005 without any other changes? For coco 2017 trainset, I found that training iteration with 12  epochs is just around 160k, which is half of iteration claimed in maskrcnn-benchmark. Does it means mmdetection use only half of training iterations? If not, do I need to change epoch setting from [8, 11] max 12 to [16, 22] max 24 when I use 4 gpus for training?

Thank you very much!

**Comment by liuliu66** [comment]
if convince, could you please provide a config file for "faster rcnn resnet50 fpn 1x" using 4 gpus as a reference?

**Comment by hellock** [comment]
Epoch is independent to the number of GPUs. The less gpus you use, the more iterations are required to reach the same epoch. You just need to change the learning rate, and other settings just remain the same.

**Comment by ZhangYumi** [comment]
> Epoch is independent to the number of GPUs. The less gpus you use, the more iterations are required to reach the same epoch. You just need to change the learning rate, and other settings just remain the same.

Hi，I have the same questions with @liuliu66 .I have only 2 gpus, and I have changed the learn rate to 0.005 by autoscale_lr argument. but I found the loss was still decreasing when my epoch reached 15. So I don't understand what is meaning that epoch is independent to the number of GPUs.Do you mean that the number of iterations of per epoch will be auto changed according to the number of GPUs?
